dali/util/std_cufile.h

```diff
@@ -33,6 +33,8 @@ class StdCUFileStream : public CUFileStream {
   explicit StdCUFileStream(const std::string& path);
   void Close() override;
   shared_ptr<void> Get(size_t n_bytes) override;
+  size_t ReadGPUThread(uint8_t* gpu_buffer, size_t n_bytes, \
```

Comment:
```suggestion
  size_t ReadGPUThread(uint8_t* gpu_buffer, size_t n_bytes,
```
I believe it is not necessary.

Reply:
fixed